### DISCUSSION TITLE
Example proposal with template from handbook.

### DISCUSSION THREAD

**Discussion by luyzdeleon** [pokt-network/organization#82] [Proposals]
### Proposal Title 
Project Management and Audit Trail Enhancement Proposal
### Proposer
Luis: CEO
### Date(s)
09/02/1970
### Category
- [x] Infrastructure
- [x] Protocol/Blockchain Dev
- [x] App Solutions
- [x] Quality Assurance
- [X] Organization
### Contract
- [X] This process follows the Milestone Rank Vote Framework as discussed in the Pocket Network Corporation's `Organization Handbook`
- [ ] This process `DOES NOT` follow the Milestone Rank Vote Framework as discussed in the Pocket Network Corporation's `Organization Handbook`

If `DOES NOT` follow the Handbook's Milestone Rank Vote Framework, please describe in detail why below:
[Detailed Explanation]

### Estimated Timeline
[X Units of Time]

**Evidence and Justification for this timeline:**

[Evidence and Justification]

**Timeline Consultants**

[Name:Subject]
 - Consultant 1:Infra Consultant
 - Consultant 2:QA Consultant
### How to test functionality?
[Detailed description on how to test the milestone]
### Project Abstractions
- Project 1
  - Details
- Project 2
  - Details
### QA Planification
[Detailed description of the QA plan]
### Community Buy In Evidence
[Links, quotes, evidence to support this proposal]
### Communication Plan
[How will this milestone be communicated? Describe in detail]

### General Notes
[Any notes]
### External Links
[If applicable]
### Executive Signoff
- [x] APPROVED
- [ ] DECLINED

If `DECLINED`, please describe in detail why below:

**Comment by luyzdeleon** [comment]
THIS IS A TERRIBLE IDEA

**Comment by oten91** [comment]
no please

**Comment by edjroz** [comment]
Just stahp
